src/index.tpl.html

```diff
@@ -6,6 +6,17 @@
     <title>Anti-Slavery Manuscripts</title>
     <script src="https://ft-polyfill-service.herokuapp.com/v2/polyfill.min.js?features=default,Array.prototype.includes,Array.prototype.keys"></script>
 
+    <script>
```

Comment:
You know what? This seems cool. I wouldn't be worried about minifying it, because it's third party code and it's good to have eyes on what we added.